File: trunk/Java/OPSJLib/src/ops/DefaultOPSConfigImpl.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package ops;

import configlib.ArchiverInOut;
import java.io.IOException;

/**
 *
 * @author angr
 */
public class DefaultOPSConfigImpl extends OPSConfig
{

    public DefaultOPSConfigImpl()
    {
        appendType("DefaultOPSConfigImpl");
    }

    @Override
    public void serialize(ArchiverInOut archive) throws IOException
    {
        super.serialize(archive);
    }
    
}

File: trunk/C#/Ops/DefaultOPSConfigImpl.cs
///////////////////////////////////////////////////////////
//  DefaultOPSConfigImpl.cs
//  Implementation of the Class DefaultOPSConfigImpl
//  Created on:      12-nov-2011 09:25:29
//  Author:
///////////////////////////////////////////////////////////

namespace Ops 
{
	public class DefaultOPSConfigImpl : OPSConfig 
    {
        public DefaultOPSConfigImpl()
        {
            AppendType("DefaultOPSConfigImpl");
        }

        public override void Serialize(IArchiverInOut archive) 
        {
            base.Serialize(archive);
        }
	}

}